nf_wochenende.nf

```diff
@@ -212,15 +215,13 @@ process wochenende {
     bash get_wochenende.sh
```

Comment:
The old part here that doesn't show any changes is not the same as the part in `nf_integrate.nf` on dev.

For example this part is missing:  
```
   if (params.no_abra) {
       params.abra = "--no_abra"
    } else {
       params.abra = ""
    }

    if (params.no_dup_removal) {
       params.no_duplicate_removal = "--no_duplicate_removal"
    } else {
       params.no_duplicate_removal = ""
    } 

    if (params.no_prinseq) {
       params.prinseq = "--no_prinseq"
    } else {
       params.prinseq = ""
    }
```

Reply:
cheers, good catch. Those are probably the conflicts that must be resolved, see below (I hope). Likely I forgot or messed up resyncing my `colin_dev` branch from the `dev` branch.

Reply:
Hold on, that part is there in the pull request, lines 209-225 ( I just went to add it in). Please recheck ? Also, I resolved the merge conflict (very minor stuff with comments). That part should have been mentioned if it was missing.